demo/index.html

```diff
@@ -5,12 +5,16 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css">
     <script type="module" src="./demo.ts"></script>
     <style type="text/css">
+      #videocontainer {
```

Comment:
This element needs to have `position: relative;` as of the current implementation of the csai-manager, that creates an absolute positioned video element.